test/performance/modules/PCI-DSS-module-dashboard.js

```diff
@@ -0,0 +1,48 @@
+const { WAIT_TIMEOUT, SERVER_URL } = require('../common/constants');
+const logger = require('../common/logger');
+
+module.exports = async function (context, commands) {
+  // Navigate to a URL, but do not measure the URL
+  await commands.navigate(SERVER_URL);
+
+  try {
+    // Click on Kibana menu to access Wazuh App link
+    await commands.wait.bySelector('button[data-test-subj="toggleNavButton"]', WAIT_TIMEOUT)
+    await commands.click.bySelector('button[data-test-subj="toggleNavButton"]')
+    await commands.wait.bySelector('a[href$="/app/wazuh"]', WAIT_TIMEOUT)
+    await commands.click.bySelector('a[href$="/app/wazuh"]')
+    
+    //Wait for an Wazuh home page component to be loaded
+    await commands.wait.byXpath('//*[contains(@class,"euiTitle euiTitle--small euiCard__title")]//*[contains(text(),"PCI DSS")]', WAIT_TIMEOUT)
+    await commands.wait.byCondition("!isNaN(parseInt(document.querySelector('.statWithLink').innerHTML))", WAIT_TIMEOUT)
+    await commands.click.byXpath('//*[contains(@class,"euiTitle euiTitle--small euiCard__title")]//*[contains(text(),"PCI DSS")]')
+
+    // Start collecting metrics
+    await commands.measure.start('PCI DSS module - Dashboard')
```

Comment:
The metrics should start between line 17 & 18.
The test ignores some request if the metrics starts pass that point
![Screenshot from 2022-08-04 07-53-54](https://user-images.githubusercontent.com/104914131/182831371-f3c4988f-a075-4bee-a7dc-60db810555f4.
![Screenshot from 2022-08-04 07-54-32](https://user-images.githubusercontent.com/104914131/182831385-fc8b2ffc-ff2c-4d8b-b0ca-47087613437d.png)
png)